suomen-vaylat-app/src/components/gfi/GfiDownloadMenu.jsx

```diff
@@ -129,6 +129,13 @@ const GfiDownloadMenu = ({ handleGfiDownloadsMenu, handleGfiDownload }) => {
     const [selectedLayers, setSelectedLayers] = useState([]);
 
     const [downloadFormats] = useState([
+        {
+            id: 'download-format-gpkg',
+            title: '.GPKG',
+            format: 'gpkg',
```

Comment:
If changed format here , then also need change download format enum in geoportal code